web: update file explorer expand button styles

valerybugakov opened the pull request:
## Changes

- Updated file explorer toggle button styles to be consistent with the collapse button.
- Added top offset to the sidebar content according to [the mockups](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=954%3A8290).
- Fixed tree padding util.

## Screenshots

### File explorer expand button

#### Before

<img width="415" alt="Screenshot 2021-06-17 at 10 38 03" src="https://user-images.githubusercontent.com/3846380/122352907-1cc06100-cf82-11eb-916b-301a2b8decc4.png">

#### After

<img width="427" alt="Screenshot 2021-06-17 at 10 25 39" src="https://user-images.githubusercontent.com/3846380/122351623-e209f900-cf80-11eb-8015-918b2b865d46.png">

### File explorer sidebar

#### Before

<img width="671" alt="Screenshot 2021-06-17 at 10 38 31" src="https://user-images.githubusercontent.com/3846380/122352918-21851500-cf82-11eb-9ba1-a3bafa960b62.png">

#### After

<img width="767" alt="Screenshot 2021-06-17 at 10 41 44" src="https://user-images.githubusercontent.com/3846380/122353356-8e001400-cf82-11eb-99ca-9419703c832c.png">

### File tree padding

#### Before

<img width="305" alt="Screenshot 2021-06-17 at 10 32 17" src="https://user-images.githubusercontent.com/3846380/122352213-7bd1a600-cf81-11eb-9c6d-e71475dfdb0f.png">

#### After

<img width="303" alt="Screenshot 2021-06-17 at 10 36 43" src="https://user-images.githubusercontent.com/3846380/122352613-da971f80-cf81-11eb-80b9-df35422f7dae.png">

Closes #21637.
Closes #22136.

umpox reviewed (approved):
Code LGTM. Question: Is there any more context over the padding changes? I feel like the text for directory and file names should be aligned the same

example:
![image](https://user-images.githubusercontent.com/9516420/122360471-537c8400-cf4e-11eb-93d8-8f77740728fb.png)

`.dockerignore` text should align with `ui`. WDYT?

felixfbecker commented on client/web/src/repo/RepoRevisionSidebar.tsx:
```diff
@@ -82,16 +82,10 @@ export const RepoRevisionSidebar: React.FunctionComponent<Props> = props => {
-                                isRedesignEnabled && 'px-3'
-                            )}
-                        >
+                        <div aria-hidden={true} className="d-flex repo-revision-sidebar__tabpanels explorer">
```
I just noticed this, why is the file tree and symbols sidebar hidden from screen readers? o.O

5h1rU replied:
I guess that the div was moved from the original hierarchy. The initial intention was to avoid an unnecessary `tab` (keyboard navigation) and jump straight to the tree.

felixfbecker reviewed (approved):
Nice!!

Maybe we should have a Percy snapshot for the collapsed state, wdyt?

valerybugakov commented:
Thanks, @umpox. I was confused by the parameter name `isRedesignEnabled`. Renamed it to `isDirectory` and adjusted the value a bit to align filenames with folder names.

<img width="258" alt="Screenshot 2021-06-17 at 12 45 38" src="https://user-images.githubusercontent.com/3846380/122374102-ab3dde00-cf94-11eb-8473-35f66dbe20ce.png">
